lacommunaute/notification/utils.py

```diff
@@ -27,12 +28,34 @@ def collect_first_replies():
     ]
 
 
+def collect_following_replies():
+    # vincentporte : assumed disapproved post are counted
+    return [
+        (
+            f"{settings.COMMU_PROTOCOL}://{settings.COMMU_FQDN}{topic.get_absolute_url()}",
+            topic.subject,
+            topic.poster_email,
+            f"{topic.new_replies} nouvelle réponse"
+            if topic.new_replies == 1
+            else f"{topic.new_replies} nouvelles réponses",
+        )
+        for topic in Topic.objects.filter(
+            updated__gte=last_notification(kind=EmailSentTrackKind.FOLLOWING_REPLIES), posts_count__gte=3
+        ).annotate(
+            new_replies=Count(
+                "posts",
+                filter=Q(posts__created__gte=last_notification(kind=EmailSentTrackKind.FOLLOWING_REPLIES))
+                & Q(posts__id__gt=F("first_post_id")),
```

Comment:
Si le but est bien d’exclure le premier poste, c’est plus clair écrit de la manière suivante.
```suggestion
                & ~Q(posts__id=F("first_post_id")),
```